packages/ttypescript/src/PluginCreator.ts

```diff
@@ -124,7 +124,24 @@ export class PluginCreator {
         this.validateConfigs(configs);
     }
 
-    createTransformers(params: { program: ts.Program } | { ls: ts.LanguageService }) {
+    mergeTransformers(into: ts.CustomTransformers, source: ts.CustomTransformers | TransformerBasePlugin) {
```

Comment:
Can you create new `type TransformersList = Required<ts.CustomTransformers>;` and use it as argument type

Reply:
You meant, as the return type right?

Reply:
I think we should make `mergeTransformers` as private